convert/what3words/README.md

```
@@ -0,0 +1,84 @@
+<!-- START_INFOCARD -->
+
+The `@flatfile/plugin-convert-what3words` plugin provides functionality to convert What3Words addresses to standard addresses and coordinates within Flatfile. It integrates with the official What3Words JavaScript API to perform the conversion and populate the user-specified fields with the results.
+
+**Event Type:**
+`listener.on('commit:created')`
+
+<!-- END_INFOCARD -->
+
+## Parameters
+
+#### `config.what3wordsField` - `string`
```

Markdownlint

Expected: h3; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)